src/voting-metadata-api/package.json

```diff
@@ -2,53 +2,43 @@
   "name": "ipfs-cache-api",
   "version": "1.0.0",
   "description": "API to cache and store IPFS Files on S3 as well as assist in local development",
-  "main": "src/lambda-handler.js",
   "author": "Algorand Foundation",
-  "type": "commonjs",
+  "type": "module",
+  "main": "src/lambda-handler.js",
   "scripts": {
-    "dev": "npm run routes && ts-node-dev --transpile-only --watch .env -r dotenv/config src/server.ts",
+    "audit": "check-audit",
+    "audit:resolve": "resolve-audit",
     "build": "rimraf build && tsoa spec-and-routes && tsc && copyfiles package.json package-lock.json build && cd build && npm ci --omit=dev",
-    "routes": "tsoa routes",
     "clean": "rimraf node_modules && rimraf build",
+    "dev": "npm run routes && tsx watch -r dotenv/config src/server.ts",
+    "format": "prettier --write .",
     "lint": "eslint --ext .ts,.tsx \"src/\"",
     "lint:fix": "eslint --ext .ts,.tsx \"src/\" --fix",
-    "typecheck": "tsoa routes && tsc --noEmit",
-    "format": "prettier --write .",
-    "audit:resolve": "resolve-audit",
-    "audit": "check-audit"
+    "routes": "tsoa routes",
+    "typecheck": "tsoa routes && tsc --noEmit"
   },
   "dependencies": {
+    "@makerx/node-cache": "^1.0.2",
+    "@makerx/node-ipfs": "^1.3.1",
     "algosdk": "^2.2.0",
-    "axios": "^1.3.4",
     "cors": "^2.8.5",
     "date-fns": "^2.29.3",
     "express": "^4.18.2",
-    "glob": "^9.3.2",
+    "file-type": "^19.0.0",
     "helmet": "^6.1.5",
-    "mime": "^3.0.0",
     "multer": "^1.4.5-lts.1",
-    "multiformats": "^9.9.0",
     "reflect-metadata": "^0.1.13",
     "serverless-http": "^3.2.0",
-    "sharp": "^0.32.6",
     "tsoa": "^5.1.1",
-    "tsyringe": "^4.7.0",
-    "uuid": "^9.0.0",
```

Comment:
Weird, were we not using this package?

Reply:
I think that was a sort-package-json, `tsyringe` is in package.json but `uuid` is not